Author: Lorenzo Martignoni <lorenzo.martignoni@poste.it>
Description: Customize the shorewall.conf configuration file to be more like what is
expected by Debian users.
--- shorewall.git.orig/configfiles/shorewall.conf
+++ shorewall.git/configfiles/shorewall.conf
@@ -9,7 +9,7 @@
 #		       S T A R T U P   E N A B L E D
 ###############################################################################
 
-STARTUP_ENABLED=No
+STARTUP_ENABLED=Yes
 
 ###############################################################################
 #			     V E R B O S I T Y
@@ -87,7 +87,7 @@
 
 SHOREWALL_SHELL=/bin/sh
 
-SUBSYSLOCK=/var/lock/subsys/shorewall
+SUBSYSLOCK=""
 
 TC=
 
@@ -170,7 +170,7 @@
 
 IPSET_WARNINGS=Yes
 
-IP_FORWARDING=On
+IP_FORWARDING=Keep
 
 KEEP_RT_TABLES=No
 
@@ -210,7 +210,7 @@
 
 RETAIN_ALIASES=No
 
-ROUTE_FILTER=No
+ROUTE_FILTER=Yes
 
 SAVE_ARPTABLES=No
 
